index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Intuit Accounts - Sign Up</title>
+  </head>
+  <body>
+      <header>
+          <button id="sign_in" class="but"><img src="image/lock.png" alt="lock" class="lock">Sign in</button>
+        <p><span id="space" class="gray_txt">Already have an Intuit account?</span><br>
+          <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D082e7ed1-15b6-4d9c-a5b0-c4cfc0b2a48b%26adobe_mc%3DMCMID%253D17380633903390345294292432376599021936%257CMCAID%253D2EEECCB7051596DF-600008078EA69FD9%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1574803839%26ivid%3D082e7ed1-15b6-4d9c-a5b0-c4cfc0b2a48b#">I forgot my User ID or Password Sign In</a>
+        </p>
+      </header>
+      <section id="form_header">
+        <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D07cade4b-b92b-4c5b-86ac-526257555814%26adobe_mc%3DMCMID%253D53422390496372272074315764205492827937%257CMCAID%253D2EEECDC20515F78C-600007FE4F268601%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1574813265%26ivid%3D07cade4b-b92b-4c5b-86ac-526257555814"></a><img src="image/intuit.png" alt="intuit" id="intuit">
```

Comment:
The intuit logo is not showing.

![logo](https://user-images.githubusercontent.com/31304943/69757346-1464c300-115d-11ea-870d-45b7bf1c13a6.png)